frontend/components/Course/StudentQueuePage/StudentQueue.tsx

```diff
@@ -136,18 +146,13 @@ const StudentQueue = (props: StudentQueueProps) => {
                         toastFunc={updateToast}
                     />
                 )}
-                {/* TODO: figure out this check */}
-                {/* {queue.active &&
-                    hasQuestion &&
-                    questions.length === 0 && (
-                        <Message
-                            style={{ marginTop: "10px" }}
-                            info
-                            header="Question already in queue"
-                            icon="comment alternate outline"
-                            content="You already have asked a question in another queue"
-                        />
-                    )} */}
+                <Grid.Row columns={1}>
```

Comment:
Can you wrap the entire row in a lastQuestions check here? This will prevent an empty row being rendered with nothing

Reply:
resolved